### PR TITLE
Add create contact UI

### LINKED ISSUE #5: Create a new contact UI

### PR DESCRIPTION
**Pull request by vivekmiyani**
Fixes #5.

### CODE REVIEW

**Comment by vivekmiyani** [comment]
@karthiknmenon 

image

For some reason intellisense not working particularly for BB's neetoui libraries in VS Code. Is my wheel setup broken? Or this is how it supposed to work?.

However other libraries like Formik its working as expected.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Pane, Typography } from "@bigbinary/neetoui/v2";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx:3]: Please use aliases.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import { FORM_INITIAL_VALUES } from "./constants";
+import Form from "./Form";
+
+const CreatePane = ({ isPaneOpen, setIsPaneOpen }) => {
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx:8]: We don't need to suffix/prefix component, file, folder names with `Pane, Modal, Form, Page` unless really necessary. 
In this scenario, we can have `Contacts/Create.jsx`. And the component can be named `CreateContact` or `Create`.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+
+import { Button, Pane, Toastr } from "@bigbinary/neetoui/v2";
+import { Input, Select } from "@bigbinary/neetoui/v2/formik";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:4]: Please use aliases.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      >
+        {({ isSubmitting, handleSubmit }) => (
+          <Form className="w-full">
+            <Pane.Body className="space-y-6">
+              <div className="flex flex-grow-0 space-x-3 w-full">
+                <Input
+                  label="First Name"
+                  name="first_name"
+                  placeholder="Enter First Name"
+                  required
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:39]: Do we need required here? And also in all the other occurrences?

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    await sleep(1000);
+
+    onClose();
+    Toastr.success("Contact added successfully.");
+  };
+
+  return (
+    <>
+      <Formik
+        initialValues={initialValues}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:25]: Please move `initialValues` and `validationSchema` to the scope of this directory.

**Review by karthiknmenon — COMMENTED** [review]
@vivekmiyani Please check the suggestions.

**Pull request merged**